org.eclipse.jdt.core/search/org/eclipse/jdt/internal/core/search/indexing/IIndexConstants.java

```diff
@@ -21,6 +21,7 @@ public interface IIndexConstants {
 	char[] REF= "ref".toCharArray(); //$NON-NLS-1$
 	char[] ANNOTATION_REF= "annotationRef".toCharArray(); //$NON-NLS-1$
 	char[] METHOD_REF= "methodRef".toCharArray(); //$NON-NLS-1$
+	char[] METHOD_REF_UNRESOLVED= "methodRefUnresolved".toCharArray(); //$NON-NLS-1$
```

Comment:
I know not much about indexer, but it sounds strange that we add **unresolved** methods to the index, I don't see any comparable methods in the existing code.

Assuming the reference was really wrong and was just a "typo" in the code - what kind of side effects can that "wrong" reference in the index cause?